cernopendata/modules/fixtures/data/docs/delphi-news-first-release/delphi-news-first-release.json

```
@@ -0,0 +1,21 @@
+[
+  {
+    "author": "DELPHI Collaboration",
+    "body": {
+      "content": "delphi-news-first-release.md",
+      "format": "md"
+    },
+    "date_published": "2024-06-26",
+    "experiment": [
+      "DELPHI"
+    ],
+    "short_description": {
+      "content": "The DELPHI Collaboration has released the first set of its data samples for the Open Data Portal."
+    },
```

Please, add something like this to make sure that the news is listed in the home page
```suggestion
    },
    "featured": 1,
```

---

OK, done.